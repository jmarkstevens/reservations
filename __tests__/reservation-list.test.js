import 'react-native'
import React from 'react'
import { shallow } from 'enzyme'
import ReservationListComponent from '../src/components/reservation-list'

describe('<ReservationListComponent>', () => {
  const Component = <ReservationListComponent data={{ refetch: jest.fn() }} />

  describe('Structure', () => {
    it('renders correctly', () => {
      const wrapper = shallow(Component)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
