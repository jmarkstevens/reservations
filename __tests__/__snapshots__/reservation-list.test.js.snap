// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReservationListComponent> Structure renders correctly 1`] = `
<SafeAreaView
  style={
    Object {
      "backgroundColor": "#F5FCFF",
      "flex": 1,
    }
  }
>
  <Header
    title="Reservations"
  />
  <FlatList
    ItemSeparatorComponent={[Function]}
    disableVirtualization={false}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onEndReachedThreshold={2}
    renderItem={[Function]}
    scrollEventThrottle={50}
    style={
      Object {
        "flex": 1,
      }
    }
    updateCellsBatchingPeriod={50}
    windowSize={21}
  />
  <withNavigation(Footer)
    refetch={[MockFunction]}
  />
</SafeAreaView>
`;
